test_cluster_detection.py

```diff
@@ -0,0 +1,468 @@
+"""
+Testing Cluster Estimation worker.
+"""
+
+import random
+import numpy as np
+import pytest
+import sklearn.datasets
+
+from modules.cluster_estimation import cluster_estimation
+from modules import detection_in_world
+
+MIN_TOTAL_POINTS_THRESHOLD = 100
+MIN_NEW_POINTS_TO_RUN = 10
+CENTER_BOX_SIZE = 500
+RANDOM_STATE = 0
+
+
+@pytest.fixture()
+def cluster_model():
+    model_created, model = cluster_estimation.ClusterEstimation.create(
+        MIN_TOTAL_POINTS_THRESHOLD,
+        MIN_NEW_POINTS_TO_RUN,
+        RANDOM_STATE,
+    )
+    assert model_created
+    yield model
+
+
+def generate_cluster_data(n_samples_per_cluster: "list[int]", cluster_standard_deviation: int) \
```

Comment:
Make private by adding double underscore prefix.

Reply:
adding double underscore prefix causes runtime error from __function() not being recognized